kubetools/deploy/commands/remove.py

```diff
@@ -58,3 +68,7 @@ def execute_remove(build, services, deployments, jobs):
     if jobs:
         with build.stage('Delete jobs'):
             delete_objects(build, jobs, delete_job)
+
+    if cronjobs:
+        with build.stage('Delete cronjobs'):
+            delete_objects(build, cronjobs, delete_cronjob)
```

Comment:
From what I can see this will delete all cronjobs associated with a project based on the filtering done [here](https://github.com/EDITD/kubetools/blob/f69e211e664d99376437d8db682c1e92581f9744/kubetools/deploy/util.py#L74-L88) - think it involves a fair bit more work to produce a UI that can only delete individual jobs, but worth looking into in the future. OOS for this PR.

Reply:
I added a comment to not forget